### PR TITLE
Update the bound app route configuration using the dynamic update endpoint

### PR DESCRIPTION
**Pull request by LittleBaiBai**
Compatible starting with SCG-VT v1.0.6.

Connects to #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,8 +25,9 @@ Once you have enough fun with the sample app, run the following script to clean
 
 Some other commands that might be helpful:
 ```bash
-./scripts/cf_deploy push    # builds and pushes frontend and backend
-./scripts/cf_deploy rebind  # unbinds and rebinds frontend and backend
+./scripts/cf_deploy push                     # builds and pushes frontend and backend
+./scripts/cf_deploy bound_app_config_update  # update bound apps' configuration with calling the update endpoint on the backing app. You will need to be a space developer to do so.
```
Comment (csterwa) [inline README.md:29]: I wonder if `dynamic_config_update` might be a friendlier name to users? Although, if user knows CF then this name makes sense too. Not a blocker from my perspective, just a suggestion/thought.

File: scripts/cf_deploy.sh [hunk 2]
```diff
@@ -75,6 +75,21 @@ unbind_all() {
...
+  sub_dir=$2
+
+  app_guid=$(cf app "$app_name" --guid)
+  echo "app guid for $app_name: $app_guid"
+  gateway_service_instance_id="$(cf service $GATEWAY_NAME --guid)"
+  gateway_url=$(cf curl /v2/service_instances/"$gateway_service_instance_id" | jq .entity.dashboard_url | sed "s/\/scg-dashboard//" | sed "s/\"//g")
+
+
+  status_code=$(curl -k -XPUT "$gateway_url/actuator/bound-apps/$app_guid/routes" -d "@$ROOT_DIR/$sub_dir/gateway-config.json" \
+       -H "Authorization: $(cf oauth-token)" -H "Content-Type: application/json" --write-out %{http_code}  --silent --output /dev/null)
```
Comment (csterwa) [inline scripts/cf_deploy.sh:89]: Do we want the output to be silent in this case? I have had situations where the output of the response was useful beyond the status code.

File: README.md (hunk 1 above)
Comment (LittleBaiBai) [reply README.md:29]: Is `dynamic` the word we are going to use in our doc to communicate this feature? If so, I'm happy to use it here. Users may not know which config we are updating given routes can also be configured at creation time, what about `dynamic-binding-update`?

Comment (csterwa) [reply README.md:29]: I like it! Yes, I think dynamic binding update is a great way to describe this approach via API. Thank you.

Comment (fifthposition) [reply README.md:29]: I would +1 putting some thought into what we call it in general--here and in docs.

**Comment by csterwa** [comment]
I am thinking that this should focus on routes update rather than the binding. I suggest that this be called `dynamic_route_config_update` now that I've thought about it some more.

File: scripts/cf_deploy.sh (hunk 2 above)
Comment (LittleBaiBai) [reply scripts/cf_deploy.sh]: 👍 Updated the output in the latest commit.

**Pull request merged**